installation/index.md

```diff
@@ -50,8 +50,8 @@ Please note that versions of Java higher than 11 are not supported at the moment
 | [Oracle Java](https://java.com/en/)         | Full openHAB support on all platforms                                                                                                                                                 | [Licensing restrictions](https://blog.takipi.com/running-java-on-docker-youre-breaking-the-law/), manual installation required on many Linux systems: [Ubuntu](https://help.ubuntu.com/community/Java), [Mint](https://community.linuxmint.com/tutorial/view/1091), and [Debian](https://wiki.debian.org/Java/Sun) are some examples |
 | [AdoptOpenJDK](https://adoptopenjdk.net)          | Open Source JDK backed by many large companies |                                                                                                                                                                                                                                                                                                                                      |
 
-Please download and install the **Java 11** version of the JVM.
-Note that openHAB 3 will run under Java 11, and you _can_ use it with openHAB 2.X as well. But be aware that although developers are working hard to make this work, there might be problems with the oldest parts of openHAB 2.x, such as some of the v1 bindings, due to non-backward compatible changes in Java 11.
+Please download and install the **Java 17** version of the JVM.
+Note that openHAB 4 will run under Java 17, and you _probably can_ use it with openHAB 2.X as well. But be aware that although developers are working hard to make this work, there might be problems with the oldest parts of openHAB 2.x, such as some of the v1 bindings, due to non-backward compatible changes in Java 17.
```

Comment:
Can you also update "11" to "17" on line 37 and line 44

> Make sure that you have an up to date Java 11 JVM platform

> Please note that versions of Java higher than 11 are not supported at the moment.

OH 2 will certainly not work with Java 17. :wink: There are some known issues like the removal of the JS engine (Nashorn) from Java 17 so I would not recommend using it with openHAB 3 either.

```suggestion
```

There is also some `java -version` output which would be nice if it was update to show Java 17 :slightly_smiling_face: , e.g.:

```
openjdk version "17.0.5" 2022-10-18 LTS
OpenJDK Runtime Environment Zulu17.38+21-CA (build 17.0.5+8-LTS)
OpenJDK 64-Bit Server VM Zulu17.38+21-CA (build 17.0.5+8-LTS, mixed mode, sharing)
```

Maybe also update the version in the openhab-cli output from 3.0.2 to 4.0.0 ?

```
Version:     3.0.2 (Build)
```